package.json

```diff
@@ -130,11 +127,15 @@
     "jest-canvas-mock": "^2.3.1",
     "lint-staged": "^11.0.0",
     "prettier": "^2.3.0",
+    "simple-icons": "^5.0.0",
     "stylelint": "^13.13.1",
     "stylelint-config-prettier": "^8.0.2",
     "stylelint-config-standard": "^22.0.0",
     "ts-jest": "^26.5.6",
     "ts-loader": "^9.0.0",
+    "vue-template-compiler": "^2.6.13",
+    "vue-server-renderer": "^2.6.13",
```

Comment:
We should keep depending on Nuxt for this, to align our used version with theirs.

If anything, we should remove the `vue` package, which was only added to placate SonarCloud, and let Nuxt handle all Vue versioning, as is intended.

Reply:
Seems reasonable. I didn't know why the vue package was there in the first place and that's why I took the updating approach, but I agree with you 👍.